### ISSUE TITLE
Usage questions

### ISSUE THREAD

**Issue by celalp** [talkowski-lab/gnomad-sv-pipeline#1]
Hi, 

Thanks for making all the code public. This is not really an issue but more of a usage question. I would like to be able to repeat some/all of the steps in another (but much smaller) disease cohort that I'm working on. However I do not have firecloud or WDL execution platform. 

Is it possible to include a step by step script/command list for the analysis? Or at least the order of the WDL files to run in each module. Even a basic order (in terms of programs and scripts) would be very helpful. 

Thanks
Alper

**Comment by yoyosef** [comment]
Hi @celalp 

I'm in the same situation,

[I'm finding reading their supplementary material useful, which includes a summary of each module, but I haven't delved deep into the code yet. ](https://www.biorxiv.org/content/10.1101/578674v1.supplementary-materialt)

![image](https://user-images.githubusercontent.com/28829926/57326338-44be1480-70da-11e9-9358-8b62f2ac6f12.png)

Thanks
Yosef

**Comment by cccnrc** [comment]
> Hi,
> 
> Thanks for making all the code public. This is not really an issue but more of a usage question. I would like to be able to repeat some/all of the steps in another (but much smaller) disease cohort that I'm working on. However I do not have firecloud or WDL execution platform.
> 
> Is it possible to include a step by step script/command list for the analysis? Or at least the order of the WDL files to run in each module. Even a basic order (in terms of programs and scripts) would be very helpful.
> 
> Thanks
> Alper

Yes, I agree, is it possible to have a master or similar to know the order and the input required?
